.github/workflows/bot-pr-updated.yml

```diff
@@ -60,3 +60,12 @@ jobs:
         with:
             name: abi
             path: abi.diff
+  wiki-check:
+    runs-on: ubuntu-latest
+    steps:
+      - check the number is existed in wiki
+        run: |
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          curl https://raw.githubusercontent.com/wiki/ginkgo-project/ginkgo/Changelog.md > Changelog.md
+          PR=\[\#${PR_NUMBER}\]\(https://github.com/ginkgo-project/ginkgo/(issues|pull)/${PR_NUMBER}\)
+          cat Changelog.md | grep -E ${PR}
```

Comment:
just make sure to disable it for tags, master and develop

Reply:
it's only open on pull request.
I only add exception for the branch `release*` (for the release branch merged into master) and label `1:ST:skip-wiki-check`